src/org/opendatakit/briefcase/ui/reused/transfer/sourcetarget/target/Central.java

```diff
@@ -85,7 +85,7 @@ public void onSelect(Container container) {
   public void decorate(JLabel label) {
     label.setText("" +
         "<html>" +
-        "URL: <a href=\"" + server.getBaseUrl().toString() + "\">" + getDescription() + "</a><br/>" +
+        "URL: <a style='text-decoration: none' href=\"" + server.getBaseUrl().toString() + "\">" + getDescription() + "</a><br/>" +
```

Comment:
Sorry to be nitpicky, but we should use double quotes for HTML tag attribute values. 

I know it's annoying to escape them :/

Reply:
Happy to change, but why do you want those ugly double quotes that use twice as much ink?